python/tank/bootstrap/resolver.py

```diff
@@ -294,9 +295,60 @@ def _get_pipeline_configurations_for_project(self, pipeline_config_name, current
             "The following pipeline configurations were found: %s" % pprint.pformat(pipeline_configs)
         )
 
+        pcs = []
```

Comment:
Transitioned back to yields and it works fine now. My issues with it came up during a midway point in shuffling code around, so I'm assuming now that other issues now resolved were causing problems for the iterator.